.eleventy.js

```diff
@@ -229,6 +230,10 @@ module.exports = function(config) {
     config.addTransform('disable-lazy-load', disableLazyLoad);
   }
 
+  if (process.env.ELEVENTY_ENV === 'prod') {
+    config.addTransform('responsive-images', responsiveImages);
```

Comment:
I feel like if we add more cheerio things we'll want to do them at once (parsing overhead). But this is probably fine for now.

Reply:
Do you mean to have one prod-transform taht combines all necessary sub-transforms in one pass?

Reply:
I'm not opposed to having one transform, but as a small complaint... The way we do imports currently is by creating the path with a template literal, which is a nuisance to dig into the source (can't click through in vscode). Not a necessary bit, but is there any chance we can consider using plain strings?

Reply:
> I feel like if we add more cheerio things we'll want to do them at once (parsing overhead). But this is probably fine for now.

Btw i don't think we necessarily can do these all at once. For instance, this transform mutates the html then pipes it on to the next transform. The next transform in the chain needs to send the new html to cheerio or it might be working with outdated data.